### PR TITLE
#1 Setup Docker for LocalStorageService

### LINKED ISSUE #1: [BE - File Storage]: Setup Docker container to run File Storage microservice.
Tip: we can look at Carl's experiments here:
https://github.com/veraPDF/veraPDF-rest/blob/integration/Dockerfile

Sub-tasks:
 - define base project structure based on Maven modules
 - setup Spring Boot app for File Storage service
 - implement simple endpoint (using Spring Actuator?) to return version info
 - wrap File Storage jar into Docker container

### PR DESCRIPTION
**Pull request by ArtemKiriutin**
Defined based project structure. Added simple endpoint to return verapdf version info.
closes #1

### CODE REVIEW

File: local-storage-service/server/pom.xml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-actuator</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <scope>test</scope>
+            <exclusions>
```
Comment (yuliya-ivaniukovich) [inline local-storage-service/server/pom.xml]: We don't need this exclusion, let's clean up

File: local-storage-service/server/src/main/resources/application.properties [hunk 2]
```diff
@@ -0,0 +1,6 @@
+verapdf.files.min-space-threshold=5GB
+verapdf.files.base-dir=${FILE_STORAGE_DIR_PATH}
+
+management.endpoints.web.base-path=/status
+server.servlet.context-path=/api
+server.port=8989
```
Comment (yuliya-ivaniukovich) [inline local-storage-service/server/src/main/resources/application.properties]: Do we need to customize port already?

File: local-storage-service/server/src/main/resources/application.properties [hunk 3]
```diff
@@ -0,0 +1,6 @@
+verapdf.files.min-space-threshold=5GB
+verapdf.files.base-dir=${FILE_STORAGE_DIR_PATH}
+
+management.endpoints.web.base-path=/status
+server.servlet.context-path=/api
```
Comment (yuliya-ivaniukovich) [inline local-storage-service/server/src/main/resources/application.properties]: Let's migrate to `yml` structure and update values as following:
```
server:
  servlet:
    context-path: /api/file
management:
  endpoints:
    web:
      base-path: /status

verapdf:
  files:
    base-dir: ${FILE_STORAGE_DIR_PATH}
    min-space-threshold: 5GB
```

Note that we first put generic Spring props and after that app specific

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <modules>
+        <module>local-storage-service</module>
+    </modules>
+    <parent>
+        <groupId>org.springframework.boot</groupId>
```
Comment (yuliya-ivaniukovich) [inline pom.xml]: @carlwilson Can we keep this parent (it comes with typical Spring dependency/plugin mngmt) or better to migrate to verapdf parent?

File: local-storage-service/server/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,6 @@
+FROM openjdk:11
+VOLUME /tmp
```
Comment (yuliya-ivaniukovich) [inline local-storage-service/server/Dockerfile]: Please investigate what VOLUME is for (to keep data I assume?). Probably we should update this value.

File: local-storage-service/server/Dockerfile [hunk 6]
```diff
@@ -0,0 +1,6 @@
...
+ADD target/local-storage-service-server.jar local-storage-service-server.jar
```
Comment (yuliya-ivaniukovich) [inline local-storage-service/server/Dockerfile]: What is the difference between ADD and COPY?

Does the first path depend on where we execute build from? Better to make it independent if possible.

File: local-storage-service/server/Dockerfile [hunk 7]
```diff
@@ -0,0 +1,6 @@
...
+ENV FILE_STORAGE_DIR_PATH "/opt/verapdf/localstorageservice/files"
```
Comment (yuliya-ivaniukovich) [inline local-storage-service/server/Dockerfile]: Please check where data is stored in typical DB container and use the similar pattern. 
I assume VOLUME should point to the same directory.

File: local-storage-service/server/Dockerfile (hunk 6 above)
Comment (ArtemKiriutin) [reply local-storage-service/server/Dockerfile]: It's better to use COPY command here, but we can't make the path independent. The official documentation on docker COPY command: COPY <src> <dest>.The <src> path must be inside the context of the build

File: local-storage-service/server/Dockerfile (hunk 5 above)
Comment (yuliya-ivaniukovich) [reply local-storage-service/server/Dockerfile]: As agreed, we'll clean this up for now, directory will be added in the next task where it is actually be needed

File: pom.xml (hunk 4 above)
Comment (yuliya-ivaniukovich) [reply pom.xml]: Let's keep it as spring boot for now, later we can update that when we start integrating with Jenkins if needed.

**Pull request merged**